HarpGuru/HarpFace/HarpFaceFragment/components/index.test.tsx

```diff
@@ -1,15 +1,25 @@
 import { useGlobal } from 'reactn'
 import 'react-native'
 import React from 'react'
+import { DegreeIds } from 'harpstrata'
 import { render } from '@testing-library/react-native'
 
 import { harpFaceProps } from '../../testResources'
+import { DisplayModes } from '../../../types'
+import { ExperienceModes } from '../../../helpers/setGlobalReactNState'
```

Comment:
As commented an many recent reviews, this ugly path must soon be fixed.

Reply:
Referenced in open issue: https://github.com/js-jslog/harpnative/issues/45#issuecomment-663618566